homeassistant/components/tado/strings.json

```diff
@@ -152,7 +152,7 @@
   "issues": {
     "water_heater_fallback": {
       "title": "Tado Water Heater entities now support fallback options",
-      "description": "Due to added support for water heaters entities, these entities may use different overlay. Please configure integration entity and tado app water heater zone overlay options."
+      "description": "Due to added support for water heaters entities, these entities may use different overlay. Please configure integration entity and tado app water heater zone overlay options. Otherwise, please configure the integration entity and Tado app water heater zone overlay options (under Settings -> Rooms & Devices -> Hot Water)."
```

Comment:
This addition is just repeating the previous sentence and adding a hint where to find the options. We don't need to repeat the previous sentence.

The language is also not really clear. "Otherwise" is used, but then we're repeating the previous instruction.